universe/package.json

```diff
@@ -2,13 +2,13 @@
   "packagingVersion": "4.0",
   "minDcosReleaseVersion": "1.9",
   "name": "cockroachdb",
-  "version": "1.0.1-1.0.5",
+  "version": "1.0.0-1.1.0",
```

Comment:
This is rather peculiar. Is `universe/package.json` metadata for DC/OS, as opposed to as a npm package?

Reply:
Yeah, it's for their Universe package manager.